### DISCUSSION TITLE
[react] interface FunctionComponent -> PropsWithChildren missing in definition version 18

### DISCUSSION THREAD

**Discussion by prashant-rajoriya-kredx** [DefinitelyTyped/DefinitelyTyped#59810] [Issues with a @types package] [answered]
### Version 18:

  `  
   interface FunctionComponent<P = {}> {
        (props: P, context?: any): ReactElement<any, any> | null;
        propTypes?: WeakValidationMap<P> | undefined;
        contextTypes?: ValidationMap<any> | undefined;
        defaultProps?: Partial<P> | undefined;
        displayName?: string | undefined;
    }
    `
    
    
    
    ### Version 17:
    
    `   interface FunctionComponent<P = {}> {
        (props: PropsWithChildren<P>, context?: any): ReactElement<any, any> | null;
        propTypes?: WeakValidationMap<P> | undefined;
        contextTypes?: ValidationMap<any> | undefined;
        defaultProps?: Partial<P> | undefined;
        displayName?: string | undefined;
    }`

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment, marked as answer]
This change was intended. Check out https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56210 for a full list of intended, breaking changes and their rationale. The PR includes links to a codemod that helps migrating to `@types/react@18`

Duplicate of https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/59760
